mail/mlmmj/Portfile

```diff
@@ -0,0 +1,30 @@
+PortSystem          1.0
+
+name                mlmmj
+version             1.4.6
+categories          mail
+platforms           darwin
```

Comment:
Could you please drop this line, it is the default. We only need to specify platforms if they are different from `darwin`.